catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/metacard/notes/NoteMetacard.java

```diff
@@ -0,0 +1,45 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.catalog.ui.metacard.notes;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import ddf.catalog.data.MetacardType;
+import ddf.catalog.data.impl.MetacardImpl;
+import ddf.catalog.data.types.Core;
+
+public class NoteMetacard extends MetacardImpl {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(NoteMetacard.class);
+
+    /**
+     * Constructor, creates a Note Metacard.
+     * @param metacardType A metacardType NoteMetacardType
+     * @param id The parent ID of the note
+     * @param comment The comment itself
+     */
+    public NoteMetacard(final MetacardType metacardType,
+            final String id,
+            final String user,
+            final String comment) {
+        super(metacardType);
```

Comment:
❓ Should we be statically setting the type of NoteType in this implementation instead of passing in a type to set every time? (The type should not be changing in this case, so we should be able to set it without passing it in)

Reply:
changed to static, which meant creating a proper NoteMetacardType class and pulling out the corresponding definition from blueprint.